packages/help-center/src/components/help-center-container.tsx

```diff
@@ -39,6 +39,7 @@ const HelpCenterContainer: React.FC< Container > = ( {
 	currentRoute,
 	openingCoordinates,
 } ) => {
+	const [ closed, setClosed ] = useState( true );
```

Comment:
I think we should call this `closing` to convey the transitory nature of this state.

Reply:
Very true, I switched it to `renderContainer` as I think that better describes what its controlling.

Reply:
I am open to changing it, but I would say that this isn't true while transitioning. It is only true after the transitions have finished. To make it something gerund I feel would imply that it changes during the transition. If anything I could see the `show` in this instance being the variable that needs to demonstrate transitory since it changes prior to actually removing it from the DOM.